### PR TITLE
Fixes for login plugins

### PR DESCRIPTION
**Pull request by dragos-dumi**

### CODE REVIEW

File: src/Form/EntryForm.php [hunk 1]
```diff
@@ -117,9 +130,21 @@ public function submitForm(array &$form, FormStateInterface $form_state) {
...
+      foreach ($this->tfaLoginPlugins as $plugin) {
+        if (method_exists($plugin, 'submitForm')) {
+          $plugin->submitForm($form, $form_state);
+        }
+      }
+    }
+
     user_login_finalize($user);
 
+    // TODO Should finalize() be after user_login_finalize or before?!
```
Comment (nerdstein) [inline src/Form/EntryForm.php:145]: I'm not sure, but let's try this and possibly note it in an issue to explore it moving fwd.

File: src/TfaLoginPluginManager.php [hunk 2]
```diff
@@ -26,9 +29,36 @@ class TfaLoginPluginManager extends DefaultPluginManager {
...
+  public function createInstance($plugin_id, array $configuration = array()) {
+    return parent::createInstance($plugin_id, $configuration); // TODO: Change the autogenerated stub
+  }
+
+  /**
+   * Returns an array of enabled login plugins.
+   * @param array $configuration
+   * @return array|null
+   */
+  public function getPlugins($configuration = array()){
```
Comment (nerdstein) [inline src/TfaLoginPluginManager.php:53]: Do we need this?

I wrote this for password policy and we didn't use it - https://github.com/d8-contrib-modules/password_policy/blob/master/src/PasswordConstraintPluginManager.php

Comment (nerdstein) [inline src/TfaLoginPluginManager.php:53]: I think you actually use functions in the default plugin manager(e.g. getInstances) to retrieve the plugins

**Comment by nerdstein** [comment]
@dragos-dumi - I sent you an invite so you could join our collaboration group. Please accept it, so you can add issues, assign yourself issues, and it will be clear what you are working on for the other participants.

File: src/TfaLoginPluginManager.php (hunk 2 above)
Comment (dragos-dumi) [inline src/TfaLoginPluginManager.php:53]: I'm using getPlugins to get the selected plugins and use them in buildForm. I had no other idea on how to do that. see #29 - maybe I should redo this pull req in other way. Thx

**Comment by nerdstein** [comment]
Merging.

**Pull request merged**